.github/workflows/binaries.yml

```diff
@@ -0,0 +1,61 @@
+name: Generate binaries
+on:
+  release:
+    types:
+      - created
+jobs:
+  binaries:
```

Comment:
This might see a bit "manual", but I've tried some other Github Actions that exist out there and they're mostly created to do cross-compiling in Go without needing CGO. So, I got actions that refused to run in Windows machines and things like that.

In any case, those actions aren't doing anything magical... I ended up doing all this manually which allow us to be more flexible anyway.

Reply:
Sounds good!